plugins/database/mysql/mysql.go

```diff
@@ -316,6 +264,99 @@ func (m *MySQL) RotateRootCredentials(ctx context.Context, statements []string)
 	return m.RawConfig, nil
 }
 
+// SetCredentials uses provided information to set the password to a user in the
+// database. Unlike CreateUser, this method requires a username be provided and
+// uses the name given, instead of generating a name. This is used for setting
+// the password of static accounts, as well as rolling back passwords in the
+// database in the event an updated database fails to save in Vault's storage.
+func (m *MySQL) SetCredentials(ctx context.Context, statements dbplugin.Statements, staticUser dbplugin.StaticUserConfig) (username, password string, err error) {
+	if len(statements.Rotation) == 0 {
```

Comment:
May not be necessary since `Rotation` is a new item in the struct, but the other functions call `statements = dbutil.StatementCompatibilityHelper(statements)`. Is this necessary here too?

Reply:
Rotation statements aren't even supported in this helper and was strictly for backwards compat of old versions of the struct.  I think it is safe to ignore here.